src/js/compatibility.js

```diff
@@ -0,0 +1,79 @@
+(function(playerLibrary) {
+    // Verify JW Player is loaded
+    if (!window.jwplayer) {
```

Comment:
`if (!playerLibrary) {` would work just as well and not repeat `window.jwplayer`.